packetbeat/docs/reference/configuration/packetbeat-options.asciidoc

```diff
@@ -418,11 +418,11 @@ Example configuration:
 
 [source,yml]
 ------------------------------------------------------------------------------
-protocols:
-  http:
-    ports: [80, 8080]
-    send_response: true
-    include_body_for: ["text/html"]
+  protocols:
```

Comment:
I think `packetbeat.protocols` is better, because when people copy/paste it's more likely to get it right.